receiver/vcenterreceiver/metadata.yaml

```diff
@@ -611,3 +633,30 @@ metrics:
     gauge:
       value_type: double
     attributes: []
+  vcenter.vm.vsan.throughput:
+    enabled: false
+    description: The vSAN throughput of a virtual machine.
+    unit: "By/s"
+    gauge:
+      value_type: double
+    attributes: [vsan_throughput_direction]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.107.0"
+  vcenter.vm.vsan.operations:
+    enabled: false
+    description: The vSAN IOPs of a virtual machine.
+    unit: "{operations/sec}"
```

Comment:
Are there no counter versions of these metrics?

Reply:
@djaglowski so these are very similar to all of the performance metrics that we get. They are actually delta metrics (but we don't have full control of the delta).
With the performance metrics, they all are deltas with an interval of 20s. We had a discussion about what to do with them at one point, and converting to rates and treating as a gauge seemed more appropriate than anything else.

The vSAN metrics are all basically the same (delta with fixed interval), but the interval is 5m.